tex/generic/pgf/math/pgfmathfunctions.code.tex

```diff
@@ -184,7 +184,7 @@
             \expandafter\pgfmath@toks\expandafter=\expandafter{\pgfmath@local@body}%
         \else%
             \pgfmath@toks={}%
-            \expandafter\pgfmath@local@function@@body\pgfmath@local@args,,%
+            \expandafter\pgfmath@local@function@@body@trimspaces\pgfmath@local@args,,%
```

Comment:
`\exp_last_unbraced:No` fits this case, but it won't obviously increase the readability.